scripts/initialize.sh

```diff
@@ -33,7 +34,8 @@ help()
    echo -e "\t--beneficiary: Staking beneficiary address"
    echo -e "\t--authorizer: Staking authorizer address"
    echo -e "\t--stake-amount: Staking amount"
```

Comment:
In L25 we've got `--staking-amount`, here we have `--stake-amount`. Let's make them consistent.

Reply:
[5f0be61](https://github.com/keep-network/keep-core/pull/3096/commits/5f0be61de413f66008ee3ab59d9db5d681e2bf61)